src/Tizen.NUI/src/public/BaseComponents/TextEditor.cs

```diff
@@ -1193,6 +1209,30 @@ public InputMethodContext GetInputMethodContext()
             return inputMethodContext;
         }
 
+        /// <summary>
+        /// Select the whole text.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
```

Comment:
Could you update this comment please?
If it's not opened as public API now and [EditorBrowsable] tag is added, then you can remove /// <since_tizen> tag now. :)